Dockerfile-debian.template

```diff
@@ -124,19 +123,23 @@ RUN set -ex; \
 # gpg key from https://nextcloud.com/nextcloud.asc
     gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys 28806A878AE423A28372792ED75899B9A724937A; \
     gpg --batch --verify nextcloud.tar.bz2.asc nextcloud.tar.bz2; \
-    tar -xjf nextcloud.tar.bz2 -C /usr/src/; \
+    tar -xjf nextcloud.tar.bz2 -C /var/www/; \
+    rm -rf /var/www/html; \
+    mv /var/www/nextcloud /var/www/html; \
     gpgconf --kill all; \
-    rm -r "$GNUPGHOME" nextcloud.tar.bz2.asc nextcloud.tar.bz2; \
-    rm -rf /usr/src/nextcloud/updater; \
-    mkdir -p /usr/src/nextcloud/data; \
-    mkdir -p /usr/src/nextcloud/custom_apps; \
-    chmod +x /usr/src/nextcloud/occ; \
-    \
+    rm -r "$GNUPGHOME"; \
+    rm -rf /var/www/html/updater; \
+    mkdir -p /var/www/html/data; \
+    mkdir -p /var/www/html/custom_apps; \
+    chmod +x /var/www/html/occ; \
+    touch /var/www/html/data/.ocdata; \
```

Comment:
If we're going to break things anyway I'd also get rid of the `data` folder inside webroot.

Reply:
You mean, you want to move it somewhere else?
I'm fine, but this `.ocdata` is needed for Nc to boot (AFAICR).